examples/texteditor/src/main/java/com/example/texteditor/MainActivity.kt

```diff
@@ -21,12 +31,14 @@ class MainActivity : AppCompatActivity() {
             }
         }
     }
+    private val peerSelectionInfos = mutableMapOf<ActorID, PeerSelectionInfo>()
 
     override fun onCreate(savedInstanceState: Bundle?) {
         super.onCreate(savedInstanceState)
         binding = ActivityMainBinding.inflate(layoutInflater)
         setContentView(binding.root)
         binding.textEditor.textEventHandler = viewModel
+        viewModel.configurationChanged = savedInstanceState != null
```

Comment:
since there is possibility of remote events getting lost during the process of configuration change, it would be better to disable the view's automatic data restoration by Android and initialize the EditText's text with `Document` on `onCreate`.

It is also possible to change shared flow to channel to guarantee all events to be collected, but considering the characteristics of a real-time collaboration, I think using `Document` as a SSOT is the saftest way to handle this situation.